clang/test/CodeGenSYCL/accessor_no_alias_property.cpp

```diff
@@ -0,0 +1,19 @@
+// RUN: %clang_cc1 -fsycl-is-device -triple spir64-unknown-unknown-sycldevice -emit-llvm %s -o - | FileCheck %s
+
+// CHECK: define {{.*}}spir_kernel void @_ZTSZ4mainE15kernel_function({{.*}} noalias {{.*}} %_arg_, {{.*}})
+
```

Comment:
Could you also add a case where the accessor has some other property and verify that the kernel *does not* have the noalias attribute?